.changeset/sour-pears-exercise.md

```diff
@@ -0,0 +1,15 @@
+---
+'@hashicorp/react-combobox': major
+---
+
+Breaking Change: inputProps now handles all input related features to allow more flexibility
+
+### What changed?
+
+Previously `onInputChange` and `onInputBlur` were the only props consumers could use to customize the behavior of the `<input />` internally.
```

Comment:
Since this package exports so many different components, could you clarify which ones are affected?